packages/guides-restructured-text/src/RestructuredText/Directives/RstClassDirective.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\RestructuredText\Directives;
+
+use phpDocumentor\Guides\Nodes\ClassNode;
+use phpDocumentor\Guides\Nodes\DocumentNode;
+use phpDocumentor\Guides\Nodes\Node;
+use phpDocumentor\Guides\RestructuredText\Nodes\CollectionNode;
+use phpDocumentor\Guides\RestructuredText\Parser\Directive;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+
+use function array_map;
+use function array_merge;
+use function explode;
+
+/**
+ * This is an alias of the `.. class::` directive
+ *
+ * When the default domain contains a class directive, this directive will be shadowed. Therefore, Sphinx re-exports it as rst-class.
+ *
+ * See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rstclass
+ */
+class RstClassDirective extends ClassDirective
+{
+    public function getName(): string
+    {
+        return 'rst-class';
```

Comment:
There is no need for an extra class, as directives support the getAliases() method. You can simply add an alias there.

Reply:
yeah right, forgot about that one, thanks!